switch_model/wecc/get_inputs.py

```diff
@@ -428,54 +434,41 @@ def query_db(full_config, skip_cf):
             "trans_length_km",
             "trans_efficiency",
             "existing_trans_cap",
+            "trans_dbid",
+            "trans_derating_factor",
+            "trans_terrain_multiplier",
+            "trans_new_build_allowed",
         ],
         """
          SELECT start_load_zone_id || '-' || end_load_zone_id, t1.name, t2.name,
-             trans_length_km, trans_efficiency, existing_trans_cap_mw
+             trans_length_km, trans_efficiency, existing_trans_cap_mw, transmission_line_id,
+            derating_factor, terrain_multiplier * transmission_cost_econ_multiplier as terrain_multiplier,
+            new_build_allowed
          FROM switch.transmission_lines
              join load_zone as t1 on(t1.load_zone_id=start_load_zone_id)
              join load_zone as t2 on(t2.load_zone_id=end_load_zone_id)
-         WHERE start_load_zone_id <= end_load_zone_id 	
+         WHERE start_load_zone_id < end_load_zone_id
```

Comment:
Is there a reason why to remove the <= sign from this?

Reply:
Yes, because there wouldn't be lines from LZ_i to LZ_i. So <= or < would be the same :)